packages/react/src/components/AccountSettings/ChangePassword/__tests__/ChangePassword.test.tsx

```diff
@@ -98,7 +110,129 @@ describe('ChangePassword', () => {
 
     fireEvent.submit(submitButton);
 
-    // submit handling is async, wait for error to be displayed
-    await waitFor(() => expect(screen.findByText('Mock Error')).toBeDefined());
+    // findBy is already async, so we do not need to use `waitFor`
+    // https://testing-library.com/docs/queries/about#types-of-queries
```

Comment:
#TIL 

This comment makes a lot of sense when reading this PR, but if without the previous code, the reader may be confused why you need to mention `waitFor` here. Maybe move it to a PR comment instead?

Reply:
Agreed! https://github.com/aws-amplify/amplify-ui/pull/2900/commits/11593c3ef3679fcb4f485914940d2b3249592342